shop/src/main/java/ro/msg/learning/shop/controller/test/ProductOrderTestController.java

```diff
@@ -0,0 +1,132 @@
+package ro.msg.learning.shop.controller.test;

```

Comment:
What? very weird flex, but ok. This looks as an attempt to populate the db with info special for the testing. :) Its a cute approach, kudos for imagination. However, all test data should live in the test file. The saving and deleting of data could be made in the required test classes using a BeforeEach/Before method and After method. Also in this stage of testing, you could just as well only use mocks, as we are unit testing our methods and not testing the calls on the db. Calling both these methods before and after is very rudimentary, but imaginative
![guy-reinvents-the-wheel-cuts-it-in-half-to-make-a-fully-functional-bike-192101-7](https://user-images.githubusercontent.com/91123200/182912222-1493ae0b-5fb1-407a-851c-72ac17121f85.jpg)
.